src/video/uikit/SDL_uikitvideo.m

```diff
@@ -233,6 +238,16 @@ CGRect UIKit_ComputeViewFrame(SDL_Window *window, UIScreen *screen)
 
     return frame;
 }
+#else
+CGRect UIKit_ComputeViewFrame(SDL_Window *window){
+    SDL_UIKitWindowData *data = (__bridge SDL_UIKitWindowData *)window->driverdata;
+#if !TARGET_OS_XR
```

Comment:
`#if TARGET_OS_XR`...

Reply:
Actually, isn't this code already in the TARGET_OS_XR case after the !TARGET_OS_XR case above?
This is one of the reasons why I like to have `#if X ... #else ...`, because it makes these kinds of things easier to spot.